Makefile

```diff
@@ -1,157 +1,73 @@
-EXEC_DIR = executables/
-TAPE_DIR = Images/tapes/
-GIFS_DIR = Images/gifs/
-PACKAGE_NAME = tcping
-VERSION = 2.6.0
-ARCHITECTURE = amd64
-DEB_PACKAGE_DIR = $(EXEC_DIR)debian
-DEBIAN_DIR = $(DEB_PACKAGE_DIR)/DEBIAN
-CONTROL_FILE = $(DEBIAN_DIR)/control
-EXECUTABLE_PATH = $(EXEC_DIR)/tcping
-TARGET_EXECUTABLE_PATH = $(DEB_PACKAGE_DIR)/usr/bin/
-PACKAGE = $(PACKAGE_NAME)_$(ARCHITECTURE).deb
-MAINTAINER = https://github.com/pouriyajamshidi
-DESCRIPTION = Ping TCP ports using tcping. Inspired by Linux's ping utility. Written in Go
-
-.PHONY: all build update clean format test vet gitHubActions container \
-		create_dirs build_linux_amd64 build_debian_package build_linux_static build_linux_arm \
-		build_windows build_macos build_macos_arm  build_freebsd
+# ==================================================
+# Constants
+# ==================================================
+
+# Meta
```

Comment:
Running the `make release` as is results in:

```console
[+] Creating debian package: output/tcping-amd64.deb
/bin/sh: 5: pushd: not found
/bin/sh: 14: popd: not found
```

Needed this in the Makefile:

```suggestion
# Meta
SHELL := /bin/bash
```

Reply:
Oh yeah, forgot about that. Bash is my default shell so it worked for me.

Out of curiosity, what shell are you using?

Reply:
Fish. But I tried Bash and it failed there too.

Reply:
Curious. I'm guessing `/bin/sh` symlinks to fish on your box?

Reply:
it is symlinking to `Dash`

Reply:
Although I tried this on two different machines - ZorinOS and Ubuntu.

Do not have access to the Ubuntu machine till Monday

Reply:
> it is symlinking to `Dash`

Yeah that makes sense. `make` uses `/bin/sh` by default. So even if your active shell is bash, it will still launch a dash subshell if `/bin/sh` is symlinked to that.

Reply:
Good to know